test/e2e/framework/kcp.go

```diff
@@ -850,6 +850,8 @@ func NewFakeWorkloadServer(t *testing.T, server RunningServer, org logicalcluste
 	require.NoError(t, err)
 	kubefixtures.Create(t, crdClient.ApiextensionsV1().CustomResourceDefinitions(),
 		metav1.GroupResource{Group: "apps.k8s.io", Resource: "deployments"},
+		metav1.GroupResource{Group: "core.k8s.io", Resource: "services"},
```

Comment:
This should probably be cleaned up and managed in a distinct PR.

Reply:
we have to create it otherwise e2e on fake downstream cluster will fail.

Reply:
Yeah, I'm currently looking to take care of this in a distinct PR. Let's keep it here if your PR is merged before.

Reply:
Done in PR https://github.com/kcp-dev/kcp/pull/2379/files#diff-6119c8a3bef16c15807ae827fde6e0f6b9e6d893969064b4cc24063755a90fffR848
Only `ingresses` are not added, but I don't see why we would need `ingresses` in all cases.

Reply:
Is there a problem when the ingress is not there ?

Reply:
I see, sure. Thanks for the answer.